Recipes/Recipes/ValidateUserData.swift

```diff
@@ -0,0 +1,62 @@
+// ValidateUserData.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Валидация введенных данных пользователя
+protocol DataValidable: AnyObject {
```

Comment:
```suggestion
protocol DataValidableProtocol: AnyObject {
```
у делегата по гайду в конце Protocol